demos/common/python/models/centernet.py

```diff
@@ -51,7 +52,14 @@ def preprocess(self, inputs):
         scale = max(height, width)
         trans_input = self.get_affine_transform(center, scale, 0, [self.w, self.h])
         resized_image = cv2.warpAffine(image, trans_input, (self.w, self.h), flags=cv2.INTER_LINEAR)
-        resized_image = np.transpose(resized_image, (2, 0, 1))
+        if self.is_onnx_format and self.reverse_input_channels:
+            resized_image = cv2.cvtColor(resized_image, cv2.COLOR_BGR2RGB)
+
+        resized_image = resized_image.transpose((2, 0, 1))  # Change data layout from HWC to CHW
+        resized_image = resized_image.reshape((self.n, self.c, self.h, self.w))
+
+        if self.is_onnx_format and self.mean_values is not None:
```

Comment:
If `self.mean_values` is `None`, `self.scale_values` is silently discarded.

Reply:
I think condition is correct, because `self.scale_values` are used only if `self.mean_values` are set.

Reply:
Yes, that's what happens, but it can be unexpected. The only relation of `scale_values` and `mean_values` you mention is `Division is applied after mean values subtraction.` It is ambiguous what happens if `mean_values` is None. One may assume `scale_values` is still applied.

Reply:
Just realized it can be tempting for you now to only update a help message :) But raising exception or applying `scale_values` is better. Exceptions can't be ignored compared to help messages by a reader. And applying only `scale_values` is what MO does, I assume. I think we should mimic MO.